models/comment.model.js

```diff
@@ -5,12 +5,12 @@ const CommentSchema = Schema({
     type: String,
     required: true,
   },
-  userID: {
+  userIDcomment: {
```

Comment:
Better to avoid using entity name for schema keys.
```suggestion
  userID: {
```